src/Symfony/Component/Yaml/Tests/Fixtures/sfMergeKey.yml

```diff
@@ -43,11 +43,11 @@ yaml: |
 php: |
     array(
         'foo' => array('a' => 'Steve', 'b' => 'Clark', 'c' => 'Brian'),
-        'bar' => array('a' => 'before', 'd' => 'other', 'b' => 'new', 'c' => array('foo' => 'bar', 'bar' => 'foo'), 'x' => 'Oren'),
+        'bar' => array('a' => 'before', 'b' => 'new', 'c' => array('foo' => 'bar', 'bar' => 'foo'), 'd' => 'other', 'x' => 'Oren'),
```

Comment:
What is the reason to change the fixture file? Is it because of the changed order of keys in the result you talked about in the issue description? If so, I suggest that we look for how to adjust the test instead. If there is not a solution, I would not make this change as it could mean that people rely on this too and we would break the applications then.